src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -47,19 +47,26 @@ public static X509Certificate2Collection GetClientCertificateCollection()
 
             private static X509Certificate2 GetCertificate(string certificateFileName)
             {
+                // Use a unique path for each load to avoid concurrency issue in the OS
+                // when loading the same certificate from multiple threads.
+                string tmpPath = Path.GetTempFileName();
                 try
                 {
-                    var cert = new X509Certificate2(
-                        Path.Combine(TestDataFolder, certificateFileName),
+                    File.Copy(Path.Combine(TestDataFolder, certificateFileName), tmpPath, overwrite: true);
+                    return new X509Certificate2(
+                        tmpPath,
                         CertificatePassword,
                         X509KeyStorageFlags.DefaultKeySet);
-                    return cert;
                 }
                 catch (Exception ex)
                 {
                     Debug.Fail(nameof(Configuration.Certificates.GetCertificate) + " threw " + ex.ToString());
                     throw;
                 }
+                finally
+                {
+                    try { File.Delete(tmpPath); } catch { }
```

Comment:
Why not just do File.GetAllBytes() and call the bytearray-ctor?

Reply:
Because I didn't know those ctors were equivalent :)